sparkle/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@dust-tt/sparkle",
-  "version": "0.2.29",
+  "version": "0.2.32",
```

Comment:
It looks likes we missed updating the .lock when we updated the package.json in the 2 previous versions!